bundles/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/discovery/KNXnetDiscoveryService.java

```diff
@@ -113,7 +113,7 @@ private synchronized void startDiscovery() {
                                 .withLabel(response.getDevice().getName()).withProperty("serialNumber", serial)
                                 .withProperty("type", "TUNNEL")
                                 .withProperty("ipAddress",
-                                        "" + response.getControlEndpoint().endpoint().getAddress().getHostAddress())
+                                        response.getControlEndpoint().endpoint().getAddress().getHostAddress())
```

Comment:
Usually this pattern is setup to prevent a `NPE` or force string type. Sure it is no longer needed?

Reply:
@lsiepel Lets stay on the safe side; the data comes from an external library.
I rolled back the the version before.